container/doctabs.go

```diff
@@ -372,11 +372,17 @@ func (r *docTabsRenderer) updateIndicator(animate bool) {
 		if a := r.action; a != nil {
 			selectedPos = a.Position()
 			selectedSize = a.Size()
+			if a.MinSize().Width > selectedSize.Width {
+				selectedSize = a.MinSize()
+			}
```

Comment:
Should we perhaps do it like this instead? We avoid calling MinSize() twice and I suspect it protects us if the size changes between the two function calls. Applies to the other place as well.  
```suggestion
			minSize := a.MinSize()
			if minSize.Width > selectedSize.Width {
				selectedSize = minSize
			}
```

Reply:
👍